TF_Service_and_Tooling_Unity/Assets/TouchFree/ServiceUI/Scripts/Configuration/ConfigUI/InteractionSettingsConfigUI.cs

```diff
@@ -69,8 +69,18 @@ public class InteractionSettingsConfigUI : ConfigUI
         [Space, Tooltip("List all Settings elements that relate to the interactionType.")]
         public InteractionTypeElements[] interactionTypeElements;
 
+        public GameObject[] permissionsBlockers;
+
         private void Awake()
         {
+            if (!PermissionController.hasFilePermission)
```

Comment:
This process is repeated on multiple screens, could we put it in a helper or base class so we reuse one method?